### PR TITLE
AC Automated Test - LRAC-9051, LRAC-9053 | master

### PR DESCRIPTION
**Pull request by rafaellajordao**
https://issues.liferay.com/browse/LRAC-9051
https://issues.liferay.com/browse/LRAC-9053

### CODE REVIEW

**Comment by rafaellajordao** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/ManageUsers.testcase [hunk 1]
```diff
@@ -55,12 +79,31 @@ definition {
...
+			ACSettings.goToUserManagement();
+
+			ACSettings.addUser(userInvite = "user@liferay.com");
+		}
 
+		task ("Change your permission to adm and delete this user") {
+			ACSettings.changeUserRole(
+				newRole = "Administrator",
+				permissionType = "Member",
+				userEmail = "user@liferay.com");
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/ManageUsers.testcase:96]: Hi @rafaellajordao, I think you should change the permission back to '**member**' so when you delete the user it has member permission

Comment (ViniLopes20) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/ManageUsers.testcase:96]: After changing you could make an assert to verify that the permission has been changed to the administrator

Comment (rafaellajordao) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/ManageUsers.testcase:96]: fixed!! :) @ViniLopes20

**Comment by rafaellajordao** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/ManageUsers.testcase [hunk 2]
```diff
@@ -55,12 +79,46 @@ definition {
...
+
+			ACSettings.deleteUser(userDelete = "user@liferay.com");
+
+			Alert.viewSuccessMessage();
+
+			AssertElementNotPresent(
+				key_permissionType = "Administrator",
+				key_userEmail = "user@liferay.com",
+				locator1 = "ACSettings#USER_ROLE_BY_EMAIL");
+		}
```
[the remark this answers is not in the document]
Comment (rafaellajordao) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/ManageUsers.testcase]: hi @alexcandidos, sorry, my bad!!

**Comment by rafaellajordao** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/ManageUsers.testcase (hunk 2 above)
[the remark this answers is not in the document]
Comment (alexcandidos) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/ManageUsers.testcase]: hey @rafaellajordao , no problem! ;)

**Comment by alexcandidos** [comment]
Hi @rafaellajordao , Thanks for working on it!

The code looks good to me!

I'm forwarding it.

**Comment by alexcandidos** [comment]
ci:forward

**Pull request closed without merging**